fabric/src/main/java/org/geysermc/hydraulic/fabric/FabricUtil.java

```diff
@@ -155,6 +177,11 @@ public static Path resolveFile(Collection<Path> roots, String file) {
         return null;
     }
 
+    /**
+     * Converts an {@link Iterable} to a {@link Collection}, unless it already is one.
+     * @param iterable The {@link Iterable} to convert.
+     * @return {@code iterable} if it's already a {@link Collection}, or a copy of it as a {@link Collection} otherwise.
+     */
```

Comment:
```suggestion
    /**
     * Converts an {@link Iterable} to a {@link Collection}, unless it already is one.
     *
     * @param iterable The {@link Iterable} to convert.
     * @return {@code iterable} if it's already a {@link Collection}, or a copy of it as a {@link Collection} otherwise.
     */
```